pkg/controller/machine/controller.go

```diff
@@ -61,7 +61,7 @@ func AddWithActuator(mgr manager.Manager, actuator Actuator) error {
 func newReconciler(mgr manager.Manager, actuator Actuator) reconcile.Reconciler {
 	r := &ReconcileMachine{
 		Client:        mgr.GetClient(),
-		eventRecorder: mgr.GetRecorder("machine-controller"),
+		eventRecorder: mgr.GetEventRecorderFor("machine-controller"),
```

Comment:
Should this also be s/-/_?

Reply:
I would still prefer to keep the event source to have `-` instead of `_`.